cfg/dist.js

```diff
@@ -7,22 +7,50 @@ const baseConfig           = require( './base' );
 
 
 const distConfig = merge( {}, baseConfig, {
-    output : { libraryTarget: 'commonjs2' },
+    output : { libraryTarget: 'umd' },
 
     devtool   : 'source-map',
     externals : {
-        'codemirror/mode/jsx/jsx' : { commonjs2: 'codemirror/mode/jsx/jsx' },
+        'codemirror/mode/jsx/jsx' : {
+            commonjs  : 'codemirror/mode/jsx/jsx',
+            commonjs2 : 'codemirror/mode/jsx/jsx',
+            window    : 'jsx'
+        },
         'componentDriver'         : {
+            commonjs  : 'nessie-ui/dist/componentDriver',
             commonjs2 : 'nessie-ui/dist/componentDriver',
+            'window'  : 'ComponentDriver'
         },
         'flounder/src/core/flounder' : {
+            commonjs  : 'flounder/src/core/flounder',
             commonjs2 : 'flounder/src/core/flounder',
+            'window'  : 'Flounder'
+        },
+        'nessie-ui'  : {
+            'commonjs' : 'nessie-ui',
+            'commonjs2': 'nessie-ui',
+            'window'   : 'Nessie'
+         },
+        'prop-types' : {
+            'commonjs' : 'prop-types',
+            'commonjs2': 'prop-types',
+            'window'   : 'PropTypes'
+        },
+        addons       : {
+            'commonjs' : 'nessie-ui/dist/addons',
+            'commonjs2': 'nessie-ui/dist/addons',
+            'window'   : 'Addons'
```

Comment:
I think we can remove the `window` property here too (`addons` is a `commonjs` bundle and will never exist as `window.Addons`)